### PR TITLE
Tweak glossary entry for kube-apiserver

### PR DESCRIPTION
**Pull request by sftim**
Tweak https://kubernetes.io/docs/reference/glossary/?all=true#term-kube-apiserver

- Change title to “API server” with “kube-apiserver” listed as an AKA
 - the title change also puts the entry under “A” in the glossary, rather than after “Z”
- use “control plane” rather than “master”
- other rewording

*NB* Every existing reference to this glossary entry uses the term “API server” or a variant.
[Preview](https://deploy-preview-16294--kubernetes-io-master-staging.netlify.com/docs/reference/glossary/?all=true#term-kube-apiserver)

### CODE REVIEW

File: content/en/docs/reference/glossary/kube-apiserver.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+aka:
+- kube-apiserver
 tags:
 - architecture
 - fundamental
 ---
- Component on the master that exposes the Kubernetes API. It is the front-end for the Kubernetes control plane. 
+ The API server is a component of the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} that exposes the Kubernetes API.
+The API server is the front-end for the Kubernetes control plane.
```
Comment (steveperry-53) [inline content/en/docs/reference/glossary/kube-apiserver.md]: No hyphen for front end.
If it were an adjective, we would use a hyphen or the single word frontend.

Comment (sftim) [reply content/en/docs/reference/glossary/kube-apiserver.md]: Ah yeah (was wrong in the original too). Will fix.

Comment (sftim) [reply content/en/docs/reference/glossary/kube-apiserver.md]: (fixed)

**Pull request merged**